cmd/otelcol/config/collector/fargate_config.yaml

```diff
@@ -0,0 +1,154 @@
+# This collector config file is designed for use within an ECS task.
+# The collector should run in a sidecar container within an ECS task.
+config_sources:
+  env:
+    defaults:
+      METRICS_TO_EXCLUDE: []
+      IMAGES_TO_EXCLUDE: ["quay.io/signalfx/splunk-otel-collector"]
+
+extensions:
+  health_check:
+    endpoint: 0.0.0.0:13133
+  http_forwarder:
+    ingress:
+      endpoint: 0.0.0.0:6060
+    egress:
+      endpoint: "https://api.${SPLUNK_REALM}.signalfx.com"
+  zpages:
+    endpoint: 0.0.0.0:55679
+  smartagent:
```

Comment:
This does not include `bundleDir` or `collectd` -- is this expected?

Reply:
No, I included them.

Reply:
These aren't documented and the default should work ootb with the collector image.  I think we can remove the extension entry as a whole in this config.

Reply:
Ok, I'll remove them

Reply:
you can remove component in total.  It's not referenced by the service extensions anyway.

Reply:
Ok, I'll remove extension smartagent